### PR TITLE
Update carolina.md

### PR DESCRIPTION
**Pull request by CaroFG**

### CODE REVIEW

**Comment by fharper** [comment]
You didn't ask a review yet, but I can see you created your branch from a previous branch and not main... :D

File: events/carolina.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+https://twitter.com/CarolainFG
+
+
+## Requirements
+
+- Your event is welcoming everyone, no matter where they come from, their social status, gender, orientation, beliefs, education and their mental or physical abilities;
+- Your conference provide access for wheelchair users and people with reduced mobility;
+- You have an enforced Code of Conduct (looking for one, check confcodeofconduct.com);
+- If you record my talk, the video needs to be accessible for free to everyone and cannot be sold in any capacity;
+- You want me to speak at your event because you find my content valuable for your attendees, not because you want my employer to sponsor your event.
```
Comment (react-learner) [inline events/carolina.md:34]: Love these!

Comment (CaroFG) [reply events/carolina.md:34]: It's all from Fred's requirements. I just took what fitted me. But I agree, it's great :)

Comment (fharper) [reply events/carolina.md:34]: FYI, my list is at https://github.com/fharper/speakerkit

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**